EntryScripts/AirflowEdgeNodeSetup.sh

```diff
@@ -27,7 +27,12 @@ source InstallAirflowWithDependencies.sh
 # Celery executor architecture
 source DefineAirflowEnvironmentVariables.sh $P_SSH_USERNAME $P_AIRFLOW_DATABASE $P_MYSQL_SERVER_USERNAME $P_MYSQL_SERVER_PASSWORD $P_MYSQL_SERVER_HOSTNAME $P_RABBITMQ_QUEUENAME $P_RABBITMQ_VHOST $P_RABBITMQ_USERNAME $P_RABBITMQ_PASSWORD $P_RABBITMQ_HOSTNAME $P_SA_EMAIL $P_SA_PASSWORD $P_AIRFLOW_DATABASE_BACKFILL $P_DAGS_CORE_FOLDER
 
-source InitializeAirflow.sh $P_ENVIRONMENT $P_GITUSER $P_GITPASSWORD $P_SSH_USERNAME
+# Copy script over to SSH_USER home and open read/write
+cp /var/lib/waagent/Microsoft.OSTCExtensions.CustomScriptForLinux-1.5.4/download/0/InitializeAirflow.sh /home/$P_SSH_USERNAME/
```

Comment:
>CustomScriptForLinux-1.5.4 [](start = 45, length = 26)

Any concerns that this will break when this version changes?

Although it's a bit convoluted, you can get the directory name independent of the version change by doing something like:
`ls -ld /var/lib/waagent/* | grep ^d | awk '{print $9}' | grep Microsoft.OSTCExtensions.CustomScriptForLinux-*`

Reply:
Yes, thanks, that's a great solution.